src/core_plugins/timelion/public/directives/expression_directive.html

```diff
@@ -9,6 +9,6 @@
     ng-show="functionSuggestions.isVisible"
     suggestions="functionSuggestions.list"
     selected-index="functionSuggestions.index"
-    on-click-suggestion="suggestionClickHandler"
```

Comment:
I think I added it in the template in my refactoring PR (https://github.com/elastic/kibana/pull/11285/files#diff-6bf4e46df33f0b22d200f253c6ce536cR21), but forgot to add the corresponding method in the directive.